### DISCUSSION TITLE
Fucking update have delete ALL DATA !!!

### DISCUSSION THREAD

**Discussion by tydoo** [teslamate-org/teslamate#2800] [General]
FUCK the 1.27.0 version !!!!!

Your fucking udpate have delete ALL DATA and corupt all install !!!

Pleask check BEFORE SEND update !!!!!

**Comment by cwanja** [comment]
Not sure the update has any issues. Know a handful of people who have all updated, with none losing all of their data. Realize this happened to you, and sorry to hear that, but the update is not 'corrupt' as your issue state.
